test/postman/acs-test-collection.json

```diff
@@ -824,6 +824,129 @@
 					},
 					"response": []
 				},
+				{
+					"name": "share-alfresco-proxy",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "e0ea8602-4fb5-4378-9eea-ddb04934654b",
+								"type": "text/javascript",
+								"exec": [
+									"pm.globals.get(\"url\");",
+									"",
+									"pm.test(\"searchAlfrescoProxyStatusCodeTest\", function () {",
+									"    pm.response.to.have.status(403);",
+									"});",
+									""
+								]
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "password",
+									"value": "myadmin",
```

Comment:
Is "myadmin" the password value here?
Probably, needs to be the default "admin"

Reply:
My bad :).  Fixed now.